website/static/js/keen.js

```diff
@@ -0,0 +1,143 @@
+var keen = require('keen-js');
```

Comment:
You should require jquery, since you use it below.  `var $ = require('jquery');`